internal/tls/tls_util.go

```diff
@@ -0,0 +1,72 @@
+package tls
```

Comment:
Every function here takes in an identity as the first argument. It makes me feel like this should just be in identity, as these are validation functions useful for all kinds of servers.

Reply:
Additionally, this package name collides with/ the base `tls` package. This means it will most likely need to be aliased if used (like in your test file imports); another good reason to move this to identity as identity interface methods.

Reply:
> Every function here takes in an identity as the first argument. It makes me feel like this should just be in the identity repo as methods on the identity interface, as these are validation functions useful for all kinds of servers.

moved to https://github.com/openziti/identity/pull/60